app/Actions/Place/GetPlaceCollectionAction.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Hedonist\Actions\Place;
+
+
+use Hedonist\Repositories\Place\PlaceRepositoryInterface;
+use Illuminate\Database\Eloquent\Collection;
+
+class GetPlaceCollectionAction
+{
+    protected $placeRepository;
+
+    public function __construct(PlaceRepositoryInterface $placeRepository)
+    {
+        $this->placeRepository = $placeRepository;
+    }
+
+    public function execute(): GetPlaceCollectionResponse
+    {
+        $places = $this->placeRepository->findAll();
+        $placesCollection = $this->transformCollection($places);
+
+        return new GetPlaceCollectionResponse($placesCollection);
+    }
+
+    /**
+     * Transform items fields and pulled dependent values (category name, creator name, city)
+     * from db so when we send collection to response there were no requests to db
+     *
+     * @param Collection $places
+     * @return Collection
+     */
+    private function transformCollection(Collection $places): Collection
```

Comment:
Firstly don't leave any comments as I mentioned above and don't use transformer inside action, move this to some presenter and call it inside controller